k8s/aat/ccd/ccd-case-management-web/ccd-case-management-web.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-case-management-web
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:latest-*
+spec:
+  releaseName: ccd-case-management-web
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: ccd-case-management-web
+    version: 0.0.2
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "f61ab34c-07da-42fc-ac7e-3f8cdadfd5a1"
+      image: hmcts.azurecr.io/hmcts/ccd-case-management-web:latest
```

Comment:
don't you need some environment config like where the api gateway is?

Reply:
They are all in https://github.com/hmcts/ccd-case-management-web/blob/master/charts/ccd-case-management-web/values.aat.template.yaml

Reply:
two problems there,
1. values.aat.template isn't published / used, we should probably get rid of it as flux is replacing that
2. those values need to be different for new AAT, i.e. using the internal urls for those apps